rest/route/service.go

```diff
@@ -173,6 +173,7 @@ func AttachHandler(app *gimlet.APIApp, opts HandlerOpts) {
 	app.AddRoute("/tasks/{task_id}").Version(2).Patch().Wrap(checkUser, addProject, editTasks).RouteHandler(makeModifyTaskRoute(sc))
 	app.AddRoute("/tasks/{task_id}/annotations").Version(2).Get().Wrap(checkUser, viewAnnotations).RouteHandler(makeFetchAnnotationsByTask(sc))
 	app.AddRoute("/tasks/{task_id}/annotation").Version(2).Put().Wrap(checkUser, editAnnotations).RouteHandler(makePutAnnotationsByTask(sc))
+	app.AddRoute("/tasks/{task_id}/created_ticket").Version(2).Put().Wrap(checkUser, editAnnotations).RouteHandler(makeCreatedTicketByTask(sc))
```

Comment:
Because this is related to annotations, should this be part of the annotation route family?

Reply:
What would that look like?

Reply:
something like /tasks/{task_id}/annotation/created_ticket

Reply:
discussed over slack